ui/perfherder/graphs/TestDataModal.jsx

```diff
@@ -23,6 +23,12 @@ export default class TestDataModal extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
+      // "Pinned" values to be used in dropdown menus
+      pinned: {
+        frameworks: [],
+        projects: ['autoland', 'mozilla-central', 'mozilla-beta', 'try'],
+        platforms: [],
```

Comment:
Are these needed? `platforms` and `frameworks`?

Reply:
not currently, wasn't sure if they would be in the future, but I can remove them

Reply:
That sounds good - that way we only add what we need.